tensorbay/dataset/dataset.py

```diff
@@ -52,6 +52,9 @@ def __init__(
         if bin_point_cloud_fields:
             self.bin_point_cloud_fields = list(bin_point_cloud_fields)
 
+    def __getitem__(self, key: str) -> Any:
+        return getattr(self, key)
```

Comment:
if input a wrong key, what exception should this function raise?
`KeyError` or `AttributeError`?